tests/test_all_rules.py

```diff
@@ -1299,67 +1398,74 @@ def test_investigation_guide_uses_rule_name(self):
         """Check if investigation guide uses rule name in the title."""
         errors = []
         for rule in self.all_rules.rules:
-            note = rule.contents.data.get('note')
+            note = rule.contents.data.get("note")
             if note is not None:
                 # Check if `### Investigating` is present and if so,
                 # check if it is followed by the rule name.
-                if '### Investigating' in note:
-                    results = re.search(rf'### Investigating\s+{re.escape(rule.name)}', note, re.I | re.M)
+                if "### Investigating" in note:
+                    results = re.search(rf"### Investigating\s+{re.escape(rule.name)}", note, re.I | re.M)
                     if results is None:
-                        errors.append(f'{self.rule_str(rule)} investigation guide does not use rule name in the title')
+                        errors.append(f"{self.rule_str(rule)} investigation guide does not use rule name in the title")
         if errors:
-            self.fail('\n'.join(errors))
+            self.fail("\n".join(errors))
 
 
 class TestNoteMarkdownPlugins(BaseRuleTest):
     """Test if a guide containing Osquery Plugin syntax contains the version note."""
 
     def test_note_has_osquery_warning(self):
         """Test that all rules with osquery entries have the default notification of stack compatibility."""
-        osquery_note_pattern = ('> **Note**:\n> This investigation guide uses the [Osquery Markdown Plugin]'
-                                '(https://www.elastic.co/guide/en/security/current/invest-guide-run-osquery.html) '
-                                'introduced in Elastic Stack version 8.5.0. Older Elastic Stack versions will display '
-                                'unrendered Markdown in this guide.')
+        osquery_note_pattern = (
+            "> **Note**:\n> This investigation guide uses the [Osquery Markdown Plugin]"
+            "(https://www.elastic.co/guide/en/security/current/invest-guide-run-osquery.html) "
+            "introduced in Elastic Stack version 8.5.0. Older Elastic Stack versions will display "
+            "unrendered Markdown in this guide."
+        )
         invest_note_pattern = (
-            '> This investigation guide uses the [Investigate Markdown Plugin]'
-            '(https://www.elastic.co/guide/en/security/current/interactive-investigation-guides.html)'
-            ' introduced in Elastic Stack version 8.8.0. Older Elastic Stack versions will display '
-            'unrendered Markdown in this guide.')
+            "> This investigation guide uses the [Investigate Markdown Plugin]"
+            "(https://www.elastic.co/guide/en/security/current/interactive-investigation-guides.html)"
+            " introduced in Elastic Stack version 8.8.0. Older Elastic Stack versions will display "
+            "unrendered Markdown in this guide."
+        )
```

Comment:
We should double check that when the transform occurs, its still formatted correctly.